config/locales/publications/itt_monthly_report_generator.yml

```diff
@@ -1,23 +1,51 @@
 en:
   publications:
     itt_monthly_report_generator:
+      this_cycle: This cycle
+      last_cycle: Last cycle
       candidate_headline_statistics:
         title: Candidate Headline statistics
-        this_cycle: This cycle
-        last_cycle: Last cycle
+      age_group:
+        title: Candidate statistics by age group
+        subtitle: Age Group
+      status:
         submitted:
           title: Submitted
+          application_metrics_column:
+            this_cycle: number_of_candidates_submitted_to_date
+            last_cycle: number_of_candidates_submitted_to_same_date_previous_cycle
         with_offers:
           title: With offers
+          application_metrics_column:
+            this_cycle: number_of_candidates_with_offers_to_date
+            last_cycle: number_of_candidates_with_offers_to_same_date_previous_cycle
         accepted:
           title: Accepted
+          application_metrics_column:
+            this_cycle: number_of_candidates_accepted_to_date
+            last_cycle: number_of_candidates_accepted_to_same_date_previous_cycle
         rejected:
           title: All applications rejected
+          application_metrics_column:
+            this_cycle: number_of_candidates_who_had_all_applications_rejected_this_cycle_to_date
+            last_cycle: number_of_candidates_who_had_all_applications_rejected_this_cycle_to_same_date_previous_cycle
         reconfirmed:
           title: Reconfirmed from previous cycle
+          application_metrics_column:
+            this_cycle: number_of_candidates_with_reconfirmed_offers_deferred_from_previous_cycle_to_date
+            last_cycle: number_of_candidates_with_reconfirmed_offers_deferred_from_previous_cycle_to_same_date_previous_cycle
         deferred:
           title: Deferred
+          application_metrics_column:
+            this_cycle: number_of_candidates_with_deferred_offers_from_this_cycle_to_date
+            last_cycle: number_of_candidates_with_deferred_offers_from_this_cycle_to_same_date_previous_cycle
         withdrawn:
           title: Withdrawn
+          application_metrics_column:
+            this_cycle: number_of_candidates_with_all_accepted_offers_withdrawn_this_cycle_to_date
+            last_cycle: number_of_candidates_with_all_accepted_offers_withdrawn_this_cycle_to_same_date_previous_cycle
         conditions_not_met:
           title: Offer conditions not met
+          application_metrics_column:
+            this_cycle: number_of_candidates_who_did_not_meet_any_offer_conditions_this_cycle_to_date
+            last_cycle: number_of_candidates_who_did_not_meet_any_offer_conditions_this_cycle_to_same_date_previous_cycle
```

Comment:
@avinhurry I had to change slightly the locale for the new section. I think I am reaching a more stable format now ☝️